docker_mcp/models/params.py

```
@@ -0,0 +1,195 @@
+"""Parameter models for FastMCP tool validation."""
+
+from typing import Dict, List, Optional
```

Ruff (0.12.2)

`typing.Dict` is deprecated, use `dict` instead

(UP035)